src/Hazelcast.Net/Protocol/Codecs/SqlExecuteCodec.cs

```diff
@@ -97,10 +98,21 @@ public sealed class RequestParameters
             /// Query ID.
             ///</summary>
             public Hazelcast.Sql.SqlQueryId QueryId { get; set; }
+
+            /// <summary>
+            /// Flag to skip updating phone home statistics.
+            ///</summary>
+            public bool SkipUpdateStatistics { get; set; }
+
+            /// <summary>
+            /// <c>true</c> if the skipUpdateStatistics is received from the client, <c>false</c> otherwise.
+            /// If this is false, skipUpdateStatistics has the default value for its type.
+            /// </summary>
+            public bool IsSkipUpdateStatisticsExists { get; set; }
```

Comment:
Do we really need two parameters here? Don't we have a chance to handle it with only one veriable? AFAIU, they are depended.
Or server side expecting both of them?

Reply:
No, these 2 parameters don't really make sense, but that is the code that the generator generates. And we don't want to think, we want to be able to run the generator and just accept whatever it generates. So we'd have to fix the generator, and that has impact on *all* languages (the Java code is similar to this too). For the time being... we will have to do with this.

If we fix this code in our repo, next time we generate codecs, these changes would be overriden.